plugins/build-your-own.rst

```diff
@@ -4,7 +4,12 @@ Building Custom Plugins
 When writing your own Plugin, you need to be aware that the Plugin Client is async first.
 This means that every plugin must be written with Promises. More about this later.
 
-Each plugin must implement the ``Http\Client\Plugin\Plugin`` interface.
+Each plugin must implement the ``Http\Client\Common\Plugin`` interface.
+
+.. versionadded:: 1.1
+    The plugins were moved to the clients-common package in version 1.1.
+    If you work with version 1.0, you need to require the separate `php-http/plugins` package.
```

Comment:
If you work with version 1.0, the interface is called ``Http\Client\Plugin\Plugin`` and you need to require the separate...